### DISCUSSION TITLE
Idea : Replay a Drive

### DISCUSSION THREAD

**Discussion by skreu** [jheredianet/Teslamate-CustomGrafanaDashboards#75] [Ideas]
Hi,

I would love to be able to stream (or replay-stream?) a drive. Today I can see with beautiful graphs the global trip, but I would like to be able to see live (or re-live) the position with a map, a power gauge, battery level and more but in real time.

Don't specifically need to have a curve, just be able to watch a replay-stream of power, speed, elevation, position and more. Do you think this is possible using Teslamate and Grafana ? I know the database positions gives all that information with time but didn't find a solution yet.

Thanks very much :)

**Comment by jheredianet** [comment]
Hi, it's a good idea. Actually, I was researching about a long time ago, but it's not possible with Grafana, since this tool only show static graphs from data you provide. 
What I saw is other online tools to do that, so, what we can do is export data from a Grafana dashboard and import them on one of these tools to replay-stream. 

In any case, all those tools were paid or by subscription. I'll keep researching to see if I can find any viable options or if you know of any, let me know. Thank you

**Comment by skreu** [comment]
First of all, thanks for your rapid answer !

Nice, ok I will continue to do some research on my own, I found this link (https://github.com/factrylabs/grafana-replay-controls) that could possibly fits ? (but really not optimal ?) but didn't succeed to include it on my Grafana, due to an ReferenceError (ReferenceError: Cannot access uninitialized variable.) 

Will inform you if I found anything ;)
Cheers !

<img width="1737" alt="Screenshot" src="https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/100134927/25ae6192-622b-4bcb-9da6-4ef569e78639">

**Comment by jheredianet** [comment]
It looks nice, but something similar you already have on my repo. Try the "Tracking Drives" dashboard. When you select the drive, you want to replay, you just have to move the pointer over the metrics graph (it will show all the relevant information) and the map will track the location.

![image](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/45735191/2ede619f-f9e0-417b-b132-69d8acce8305)

I was thinking in exporting the data and use an external tool to see something like this:

https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/45735191/2d73a773-ebe5-41d8-a37f-629646bbf455
